test/basic-querying.test.js

```diff
@@ -153,6 +153,26 @@ describe('basic-querying', function() {
           done();
         });
     });
+    it('should query by ids and null condition', function(done) {
+      // due to some connectors not being assigning null value by default
+      // the query is filtered by the fields which may consist null value
+      // or not exist at all
+      User.findByIds([
+        createdUsers[0].id,
+        createdUsers[1].id],
+        {fields: {name: true, vip: true}}, function(err, users) {
```

Comment:
@paulussup Yes I agree with you. Would you like to fix the test cases to go with the changes you proposed? We can just add a property `vip` with value **null** on the second user of `createdUsers` and just query that with your previous test case.

Please be sure to pull latest from your branch if you haven't already.